### PR TITLE
Better docs

### PR DESCRIPTION
**Pull request by mdartic**
Related to #4

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# and security with helmet environment variables,
+# maybe a nginx for serving front and reverse proxy to the api and the file storage.
+version: '3'
+services:
+  lck-db:
+    image: postgis/postgis:12-3.0
+    environment:
+      POSTGRES_DB: public
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: pouicpouic
```
Comment (DarkVisenya) [inline docs/getting-started.md:26]: Maybe we need to change this. It can be misinterpreted. 

If agree, we also need to update l53

Comment (mdartic) [reply docs/getting-started.md:26]: What are your thoughts ?

File: docs/.vitepress/config.js [hunk 2]
```diff
@@ -2,73 +2,69 @@ module.exports = {
...
+      children: [{
+        text: 'Key features',
+        link: 'concepts/key-features.html'
+      }, {
+        text: 'Field types',
+        link: 'concepts/field-types.html'
+      }, {
+        text: 'Block types',
+        link: 'concepts/block-types.html'
+      }]
```
[the remark this answers is not in the document]
Comment (mdartic) [reply docs/.vitepress/config.js:54]: Yep, I think we need to work more on this.

File: docs/concepts/field-types.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Field Types
+
+In LocoKit, you can manage several field types.
+
+Here is the summary of all the types implemented.
+
+**Simple**
+* **Boolean** : Checkbox field useful for true/false values
```
[the remark this answers is not in the document]
Comment (mdartic) [reply docs/concepts/field-types.md]: Fixed

File: docs/getting-started.md (hunk 1 above)
Comment (mdartic) [reply docs/getting-started.md]: Replace by `yourPostgresPassword`

**Review by DarkVisenya — APPROVED** [review]

**Pull request merged**